src/Entity/EditableOrderTrait.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Entity;
+
+use Sylius\Component\Core\OrderPaymentStates;
+
+/** @method getPaymentState() */
```

Comment:
Use `@mixin` instead

Reply:
I couldn't recall what should I use here to have an autocompletion 😅 Thanks!